app/layout.tsx

```
@@ -127,6 +127,8 @@ export default function RootLayout({
         <link rel="icon" type="image/png" sizes="16x16" href="/favicon/favicon-16x16.png" />
         <link rel="apple-touch-icon" sizes="180x180" href="/favicon/apple-touch-icon.png" />
         <link rel="manifest" href="/favicon/site.webmanifest" />
+        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.5.1/css/all.min.css" />
+        <script src="https://unpkg.com/lucide@latest"></script>
```

ESLint

[error] Synchronous scripts should not be used. See: https://nextjs.org/docs/messages/no-sync-scripts

(@next/next/no-sync-scripts)